fw/Core/Hitcon/App/TamaApp.cc

```diff
@@ -225,6 +225,8 @@ void TamaApp::OnButton(button_t button) {
           break;
         case TAMA_APP_STATE::LV_DETAIL:
           if (_tama_data.secret_level < TAMA_MAX_SECRET_LEVEL) {
+            // Note: This testing feature may no longer work because of
+            // SecretLevelFromSponsor()
             _tama_data.secret_level++;
```

Comment:
Let's remove this variable entirely, and make a __builtin_popcount by ourselves.